peopleops/index.md

```diff
@@ -83,3 +83,14 @@ Licenses and software assets can be purchased for third-party software will be u
 If you are seeking a license of software that we do not currently use, put together a short summary as to the costs associated with the licenses and describe the value that the software would provide. If it would modify the way in which the company operates, then the best place to present this is as a PR to the Handbook. With approval from the CEO and CTO you can proceed with the acquisition of the licenses.
 
 When seeking an additional license for existing software we pay for, then you just need to justify why the expenditure is justified for you to be able to access it via an e-mail. Similarly, if the CEO & CTO agree, then you can proceed with purchasing the additional license.
+
+### Workspace & Location
+
+As an all remote company we don't have offices. It is envisaged that most people will work from their home the majority of the time. Equipment & Furniture required for a home workspace should be discussed with your manager as part of the hiring process.
```

Comment:
TIL: envisaged 😄

Reply:
Felt like the right phrasing, mostly we think that people will work from home but its not a requirement or expectation, so this opens up our hiring to people that don't have the space at home.